hack/local-up-cluster.sh

```diff
@@ -1169,10 +1169,14 @@ if [[ "${START_MODE}" != "kubeletonly" ]]; then
   start_csi_snapshotter
 fi
 
+if [[ "$(uname)" != "Linux" ]]; then
+    START_MODE="nokubeproxy"
```

Comment:
it seems like it would be better to do a uname check guarding the start_kubeproxy call like we do for the kubelet, rather than overriding a START_MODE the user may have set.

Reply:
Ok! So something like this?

```
if [[ "$(uname)" == "Linux" && "${START_MODE}" != "kubeletonly" ]]; then
  if [[ "${START_MODE}" != "nokubeproxy" ]]; then
    start_kubeproxy
  fi
fi
```

Reply:
Thanks for the review! I agree with this! I have made the changes